core/deployment/src/main/java/io/quarkus/deployment/logging/LoggingResourceProcessor.java

```diff
@@ -79,11 +81,22 @@ void miscSetup(
     @BuildStep
```

Comment:
This needs to be:

```
@BuildStep(loadsApplicationClasses = true)
```

Otherwise if you try to set a custom banner (that of course needs to be placed in `src/main/resources`), the banner not found and therefore not displayed at runtime.

Reply:
It was loading the custom banner, placed inside resources ,